src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,263 @@
+    landcover_data_path = config['landCoverDataPath']
+
+    # light checking of itu,terrain and landcover data path exists
+    self.assertTrue(os.path.exists(itu_data_path), msg='ITU Data path is not configured')
+    self.assertTrue(os.path.exists(terrain_data_path), msg='Terrain Data path is not configured')
+    self.assertTrue(os.path.exists(landcover_data_path), msg='LandCover Data path is not configured')
+
+    # Whitelist user IDs.
+    for device in config['registrationRequests']:
+      self._sas_admin.InjectFccId({'fccId': device['fccId']})
+      self._sas_admin.InjectUserId({'userId': device['userId']})
+
+    # Inject the PAL records
+    for pal_record in pal_records:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record)
+
+    # Pre-load conditional registration data for N3 CBSDs.
+    if ('conditionalRegistrationData' in config) and \
+            (config['conditionalRegistrationData']):
+      self._sas_admin.PreloadRegistrationData(
+          config['conditionalRegistrationData'])
+
+    # Register devices and  check response.
+    cbsd_ids = self.assertRegistered(config['registrationRequests'])
+
+    # Trigger SAS UUT to create a PPA boundary
+    pal_ids = []
+
+    for index, pal_record in enumerate(pal_records):
+      pal_ids.append(pal_record['palId'])
+
+    ppa_creation_request = {
+        "cbsdIds": cbsd_ids,
+        "palIds": pal_ids
+    }
+    ppa_id = self._sas_admin.TriggerPpaCreation(ppa_creation_request)
+
+    self.assertIsNotNone(ppa_id)
+
+    # Notify the SAS UUT about the SAS Test Harness
+    certificate_hash = getCertificateFingerprint(self._sas._tls_config.client_cert)
+    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
+                                   'url': self._sas._base_url})
+
+    # As SAS is reset at the beginning of the test, the FAD records should contain
+    # only one zone record containing the PPA that was generated. Hence the first
+    # zone record is retrived and verified if it matches the PPA ID.
+    response = self.TriggerFullActivityDumpAndWaitUntilComplete(
+        config['serverCert'], config['serverKey'])
+    self.assertContainsRequiredFields("FullActivityDump.schema.json", response)
+    ppa_zone_data = []
+
+    # Download 'zone' dump file. 
+    for dump_file in response['files']:
+      self.assertContainsRequiredFields("ActivityDumpFile.schema.json", dump_file)
+      downloaded_file = None
+      if dump_file['recordType'] == 'zone':
+        url = str(dump_file['url']).strip()
+        downloaded_file = self._sas.DownloadFile(url)
+        ppa_zone_data.append(downloaded_file)
+
+    # Check if the retrieved FAD PPA zone record matches the PPA ID that was generated
+    # using the admin API.
+    #self.assertEqual(ppa_zone_data[0]['id'], ppa_id)
+
+    uut_ppa_geometry = ppa_zone_data[0]['zone']['features'][0]['geometry']
+    uut_ppa_shapely_geometry = utils.GeoJsonToShapelyGeometry(uut_ppa_geometry)
```

Comment:
This line useless. See comments below. The PolygonAlmostEqual routine takes directly the geojson (with the PR #487)

Reply:
Yes this   line is no longer required. This PR has been updated based on PR#487.